docs/feature-gates.md

```diff
@@ -96,6 +97,19 @@ which is closer to its origin with this feature. Refer to this
 [link](https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/)
 for more information.
 
+
+### ProxyTerminatingEndpoints
+
+`ProxyTerminatingEndpoints` enables ProxyTerminatingEndpoints support in AntreaProxy. 
+The feature ProxyTerminatingEndpoints is at beta stage in Kubernetes 1.22 (beta), and
```

Comment:
The ProxyTerminatingEndpoints feature

is -> was

Reply:
Done